projects/ontimize-web-ngx-report/src/lib/components/report-on-demand/report-on-demand/report-on-demand.component.html

```diff
@@ -1,5 +1,5 @@
 <div mat-dialog-title class="dialog-title" fxLayout="row" fxLayoutAlign="space-between center">
-  {{ 'REPORT_ON_DEMAND_TITLE' | oReportsTranslate }}
+  <h2 class="h2_title">{{ 'REPORT_ON_DEMAND_TITLE' | oReportsTranslate }}</h2>
```

Comment:
I think it is better that the class have another name as a `title`, `report_title`